examples/bq_file_load_benchmark/benchmark_tools/bucket_util.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import logging
+
+from google.cloud import storage
+
+from benchmark_tools import file_constants
```

Comment:
Example issue with tests: test_bucket_util will only work if I modify this line to 
`from bq_file_load_benchmark.benchmark_tools import file_constants`

But then, when trying to run any operations via bq_file_load_benchmark.py, I get the following error: 
`Traceback (most recent call last):
  File "bq_file_load_benchmark.py", line 21, in <module>
    from benchmark_tools import file_generator
  File "/Users/annarudy/bq_file_load_benchmark/benchmark_tools/file_generator.py", line 28, in <module>
    from benchmark_tools import bucket_util
  File "/Users/annarudy/bq_file_load_benchmark/benchmark_tools/bucket_util.py", line 22, in <module>
    from bq_file_load_benchmark.benchmark_tools import file_constants
  File "/Users/annarudy/bq_file_load_benchmark/bq_file_load_benchmark.py", line 21, in <module>
    from benchmark_tools import file_generator
ImportError: cannot import name file_generator`